client/luigi-element.d.ts

```diff
@@ -419,6 +419,69 @@ export interface LuigiClient {
   getCurrentLocale: () => string;
   linkManager: () => LinkManager;
   uxManager: () => UxManager;
+  /**
+   * Publish an event that can be listened to on the core side.
+   *
+   * Similar to LuigiClient.sendCustomMessage(...) but for WebComponent based microfrontends.
+   *
+   * @param event Custom Event to be published
+   * @returns
+   * @memberof LuigiClient
+   *
+   * @example
```

Comment:
need to have a discussion on where to put this example at.

Reply:
This would be clarified on #3903 and the publishEvent docu can then be put in LuigiClient API section with the proper label